asterisk/rootfs/usr/share/tempio/logger.conf.gtpl

```diff
@@ -14,15 +14,15 @@ console => warning,error
 console => notice,warning,error
 
 {{- else if eq .log_level "info" -}}
-console => verbose,notice,warning,error
+console => verbose(1),notice,warning,error
 
 {{- else if eq .log_level "debug" -}}
-console => verbose,debug,notice,warning,error
+console => verbose(1),debug(1),notice,warning,error
 
 {{- else if eq .log_level "trace" -}}
-console => verbose,debug,notice,warning,error
+console => verbose(2),debug(2),notice,warning,error
 
 {{- else if eq .log_level "all" -}}
-console => dtmf,fax,verbose,debug,notice,warning,error
+console => verbose(3),debug(3),*
```

Comment:
Did you test the "*"?

Reply:
No not yet. Saw it from [here](https://github.com/asterisk/asterisk/blob/master/configs/samples/logger.conf.sample#L157).